### PR TITLE
fixed the sort npe

### PR DESCRIPTION
**Pull request by xavluiz**
the sort should check if the list has more than 1 element

### CODE REVIEW

File: src/com/softwareco/intellij/plugin/managers/FileManager.java [hunk 1]
```diff
@@ -561,7 +561,11 @@ public static String getFileContent(String file) {
     public static KeystrokeCount getLastSavedKeystrokeStats() {
         List<KeystrokeCount> list = convertPayloadsToList(getKeystrokePayloads());
         if (list != null && list.size() > 0) {
-            list.sort((o1, o2) -> o2.start < o1.start ? -1 : o2.start > o1.start ? 1 : 0);
+            if (list.size() > 1) {
+                // sort since we have more than 1 element
+                list.sort((o1, o2) -> o2.start < o1.start ? -1 : o2.start > o1.start ? 1 : 0);
```
Comment (mmclead) [inline src/com/softwareco/intellij/plugin/managers/FileManager.java:566]: if we implemented a Comparator in the KeystrokeCount class, would that make it safe to execute a Collections.max() on the list with any number of elements?    Trying to find if Java Comparators are safe for use with lists of length 1 by default.  Didn't find the answer.

Comment (xavluiz) [reply src/com/softwareco/intellij/plugin/managers/FileManager.java:566]: i'll look into it

Comment (xavluiz) [reply src/com/softwareco/intellij/plugin/managers/FileManager.java]: yeah, that works. i've added a test for that as well. good suggestion and it help for readability in regards to what the sort is doing. updated

**Review by mmclead — APPROVED** [review]

File: src/com/softwareco/intellij/plugin/managers/FileManager.java (hunk 1 above)
Comment (mmclead) [reply src/com/softwareco/intellij/plugin/managers/FileManager.java]: Very nice!

**Pull request merged**